src/server/scripts/Outland/BlackTemple/boss_teron_gorefiend.cpp

```diff
@@ -435,7 +433,7 @@ class boss_teron_gorefiend : public CreatureScript
 
                 for (uint8 i = 0; i < 2; ++i)
                 {
-                    Creature* Shadow = NULL;
+                    Creature* Shadow = nullptr;
```

Comment:
These lines should be
if (Creature* Shadow = me->SummonCreature(CREATURE_SHADOWY_CONSTRUCT, CalculateRandomLocation(me->GetPositionX(), 10), me->GetPositionY(), me->GetPositionZ(), 0, TEMPSUMMON_TIMED_OR_CORPSE_DESPAWN, 0))